config/module.config.php

```diff
@@ -2,12 +2,11 @@
 
 namespace AcMailer;
 
+use AcMailer\Service\MailService;
```

Comment:
What did you add this for?

Reply:
My ide has added automatically. I can remove it. No problem

Reply:
Looks good. I'll probably test it and merge it this weekend. Thanks :-)